File: exercicios/Python/2057/2057.py
def entrada():
    n = input().split()
    s = int(n[0])
    t = int(n[1])
    f = float(n[2])
    return(s,t,f)

def main():
    s,t,f = entrada()
    ht = 0
    
    ht = s + t + f

    if ht == 24:
        ht = 0

    elif ht > 24:
        ht = ht - 24
    elif ht < 0:
        ht = ht + 24
    print('{:.0F}'.format(ht))

main()

File: exercicios/JavaScript/2057/2057.js
var input = require('fs').readFileSync('stdin','utf-8');
var lines = input.split('\n');

var x = lines.shift().split(' ');

var s = parseInt(x[0])
var t = parseInt(x[1])
var f = parseInt(x[2])

var total = s + t + f

if(total >= 24){
    total = total - 24
}
if(total < 0){
    
    total = total + 24
}
console.log(total)